java/ql/src/semmle/code/java/frameworks/jackson/JacksonSerializability.qll

```diff
@@ -62,8 +72,20 @@ library class ExplicitlyWrittenJacksonSerializableType extends JacksonSerializab
   }
 }
 
+/** A type whose values are explicitly deserialized in a call to a Jackson method. */
+private class ExplicitlyReadJacksonSerializableType extends JacksonDeserializableType {
```

Comment:
The code here currently distinguishes between serialization and de-serialization. But here it's getting mixed up. This seems like the class should just be named `ExplicitlyReadJacksonDeserializableType`. Of course, that class already exists, so maybe just add the charpred defined here as an additional disjunct in the existing class?

Reply:
Can do!

Reply:
Done!